src/Orleans.Hosting.CloudServices/Orleans.Hosting.CloudServices.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Hosting.CloudServices</PackageId>
```

Comment:
>Microsoft.Orleans.Hosting.CloudServices [](start = 15, length = 39)

I think it should be Microsoft.Orleans.Hosting.AzureCloudServices. Otherwise, it sounds as if it's for all of the cloud.